test_particles.py

```diff
@@ -0,0 +1,21 @@
+from .particles import Particles
+import pytest
+import sys
+import numpy as np 
+
+
+def test_from_file():
+    particles =  Particles.from_file(file_name='sample_config1.xyz')
+    # File includes 800 particles
+    expected_num_particles = 800
+    calculated_num_particles = particles.num_particles
+    assert (expected_num_particles == calculated_num_particles)
+
+def test_from_random():
```

Comment:
We should check that particles are within the box dimensions.

Reply:
Sure, you're right, thank you Brian...